jerry-core/ecma/builtin-objects/ecma-builtin-date-prototype.cpp

```diff
@@ -38,6 +42,55 @@
  */
 
 /**
+ * Insert leading zeros to a string of a number if needed.
+ */
+static void
+ecma_date_insert_leading_zeros (ecma_string_t **str_p, /**< input/output string */
```

Comment:
@egavrin, I think that you are right. Following current naming style, the prefix should be even `ecma_builtin_date_prototype_helper_`. But this prefix is too long. So, probably, naming scheme for built-in helpers should be somehow updated. I'm not sure how exactly. But at least, `ecma_date_`, of course, should be added.